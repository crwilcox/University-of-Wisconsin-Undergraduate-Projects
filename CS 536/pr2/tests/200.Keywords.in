bool
double
else
false
if
int
null
read
return
string
true
void
while
write