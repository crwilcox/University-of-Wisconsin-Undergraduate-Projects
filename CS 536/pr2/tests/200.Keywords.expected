1:1:BOOL
2:1:DOUBLE
3:1:ELSE
4:1:FALSE
5:1:IF
6:1:INT
7:1:NULL
8:1:READ
9:1:RETURN
10:1:STRING
11:1:TRUE
12:1:VOID
13:1:WHILE
14:1:WRITE
